### PR TITLE
Restful Web Service Added

### PR DESCRIPTION
**Pull request by melissceylan**

### CODE REVIEW

File: src/main/resources/Run.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>ETUmulatorWeb</title>
+</head>
+<body>
+<form action="/run" method="post">
+  <div>
+    Fill here.
```
Comment (RootG) [inline src/main/resources/Run.html]: You need to add an appropriate html element here in order to submit codes.

File: src/main/java/com/kasirgalabs/etumulatorweb/rest/Run.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    @GET
+    @Produces(MediaType.TEXT_HTML)
+    public Response getView() {
+        return Response.ok(getClass().getClassLoader().getResourceAsStream("Run.html")).build();
+    }
+
+    @POST
+    @Produces(MediaType.TEXT_HTML)
+    public Response runCode(@FormParam("code") String code) {
```
Comment (RootG) [inline src/main/java/com/kasirgalabs/etumulatorweb/rest/Run.java]: Run the received code in ETUmulator and return the values of registers.

**Comment by RootG** [comment]
I have updated your fork, removed unused files, codes, etc... Please update your pull request according to my inline commands.

**Comment by melissceylan** [comment]
I changed some classes that you said to me.

**Comment by RootG** [comment]
You should run `git submodule update --init` in the root of project directory to initialize ETUmulator module. Otherwise, ETUmulator folder will be empty.

**Pull request merged**